h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -942,6 +942,10 @@ public CoxPHModelPrediction predictCoxPH(RowData data) throws PredictException {
   // Transparent methods passed through to GenModel.
   //----------------------------------------------------------------------
 
+  public GenModel getModel() {
```

Comment:
Great! Now we can make public final GenModel m; private in following PR, I'll do it.

Reply:
valenad-PUBDEV-8172-make-model-private-field